GOVERNANCE.md

```diff
@@ -61,19 +61,67 @@ maintainer has already discussed the candidacy with the candidate and a
 maintainer is willing to be a sponsor by opening the pull request. The candidate
 becomes a maintainer once the pull request is merged.
 
-## Adding sub-projects
-
-Similar to adding maintainers, new sub projects can be added to containerd
-GitHub organization as long as they adhere to the CNCF
-[charter](https://www.cncf.io/about/charter/) and mission. After a project
-proposal has been announced on a public forum (GitHub issue or mailing list),
-the existing maintainers are  given five business days to discuss the new
+## Adding subprojects
+
+containerd subprojects are divided into two flavors currently: **core** and
+**non-core**. Most of the repositories within the containerd GitHub organization are
+"core" to the delivery of the containerd runtime project's releases. For example,
+the Golang implementations of the cgroups, runc, and console APIs, the ttrpc
+implementation, and various plugins like the CRI implementation. These, among a
+handful of other repositories, are all **core** and required pieces which culminate
+in officially supported and maintained releases of containerd.
+
+Non-core projects have strong affiliation with the containerd runtime, but
+operate in some ways similar to the traditional `contrib/` directory in many
+open source projects. They are most likely separately maintained, and potentially
+have a different release cycle and support guarantee than the main containerd project
+and its core subprojects. They may be written in other languages and therefore
```

Comment:
this reads a little weird

Reply:
"and support guarantee than the main containerd project
and its core subprojects."